src/assets/components/Cards/Cards.tsx

```diff
@@ -8,6 +7,28 @@ interface Props {
   onDelete: (id: string) => void;
 }
 
+interface CardsInfoProps {
+  prop1: string;
+  prop2: string;
+}
```

Comment:
sorry for confusion, that provided code snippet was just template, you feel free to name thing as you want (prop1, prop2 isnt the best variant)

Reply:
✅